### PR TITLE
Construct Chart Label

### PR DESCRIPTION
**Pull request by mcnuttandrew**
Sometimes the title prop on the axes component leaves something to be desired, so this PR adds a customizable label that you place where ever:

Before
screen shot 2018-11-13 at 4 28 03 pm

After
screen shot 2018-11-13 at 4 27 51 pm

### CODE REVIEW

**Comment by mcnuttandrew** [comment]
Addresses #883 and #542

**Comment by mcnuttandrew** [comment]
Bump for review? @benshope

File: src/plot/chart-label.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+class ChartLabel extends React.PureComponent {
```
Comment (benshope) [inline src/plot/chart-label.js]: Can't wait to bump this to 16.6 so we can use React.memo for this!

**Comment by JelleBlaauw** [comment]
Please add support for x and y (in pixels) instead of percentages.

**Pull request merged**